.lgtm.yml

```diff
@@ -29,7 +29,8 @@ extraction:
             # setup the venv and install packages
             - "python3 -m venv ./fprime-venv"
             - ". ./fprime-venv/bin/activate"
-            - "pip install fprime-tools fprime-gds"
+            - "pip install urllib3 setuptools_scm"
```

Comment:
Same as my above comment

Reply:
No, these dependencies are of the setup.py file, and must exist before pip is run. The fix is to not use them.  That will is already fixed but will come in when we publish fpp version 1.0.2.